soc/arm64/nxp_layerscape/ls1046a/mmu_regions.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <devicetree.h>
+#include <soc.h>
+#include <sys/util.h>
+#include <arch/arm64/arm_mmu.h>
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 0),
+			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 0),
+			      MT_DEVICE_nGnRnE | MT_RW | MT_NS),
```

Comment:
Use one of `MT_P_RW_U_RW` instead of simply `MT_RW`.

Also any reason to not use `MT_DEFAULT_SECURE_STATE`?

Reply:
No special reason, will update it, thanks.